tap_gong/streams/interaction_stats.py

```diff
@@ -86,8 +87,7 @@ def validate_response(self, response: requests.Response) -> None:
             if response.status_code == 400 and not self.retried:
                 self.retried = True
                 raise RetriableAPIError(msg, response)
-            # 404 = nothing found for user - move on
-            if response.status_code != 404:
```

Comment:
separate import step function is generated for each stream, so no point of moving on. Also, other streams raise error instead of moving on.